### PR TITLE
Pol 467 disable 45 day creds

### PR DESCRIPTION
**Pull request by XOmniverse**
### Description

Verified working policy

### CODE REVIEW

**Review by rshade — DISMISSED** [review]
LGTM

File: security/aws/iam_disable_45_day_creds/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## v2.1
```
Comment (rshade) [inline security/aws/iam_disable_45_day_creds/CHANGELOG.md:3]: remove this one por favor

**Review by rshade — CHANGES_REQUESTED** [review]

File: security/aws/iam_disable_45_day_creds/CHANGELOG.md (hunk 1 above)
Comment (XOmniverse) [reply security/aws/iam_disable_45_day_creds/CHANGELOG.md:3]: You got it

**Review by rshade — APPROVED** [review]
LGTM

**Pull request merged**